src/ring_theory/witt_vector/basic.lean

```diff
@@ -257,4 +259,19 @@ def ghost_equiv : 𝕎 R ≃+* (ℕ → R) :=
 lemma ghost_map.bijective_of_invertible : function.bijective (ghost_map : 𝕎 R → ℕ → R) :=
 (ghost_equiv p R).bijective
 
+end invertible
+
+/-- `witt_vector.coeff x 0` as a `ring_hom` -/
+@[simps]
+def zeroth_coeff : 𝕎 R →+* R :=
```

Comment:
For polynomials and powerseries, we've called this `constant_coeff`. I think it makes sense to do the same here.